### DISCUSSION TITLE
Desenvolvedor consegue ter uma vida saudável?

### DISCUSSION THREAD

**Discussion by Marcelosilva10** [frontendbr/forum#752] [Discussão]
Bom dia Pessoas tudo bem com você master dev ninjas, minha dúvida é o seguinte parace uma coisa tola, mais tive essa dúvida.
Será que um desenvolvedor consegue ter uma vida Saudável com essa rotina diária na nossa área, é bem corrido por sinal com várias demandas, várias tecnologias que conseguimos aprender enfim, são muitas novidades no dia a dia.

Será que consegue ter um vida sei lá fitness kkkkk.
ótimo dia a todos e um excelente final de semana para vocês!

Quem disse que programador não pode ter uma vida SAÚDAVEL!

Vi esse artigo falando sobre isso.
[https://blog.umbler.com/br/vida-saudavel-para-desenvolvedores-e-possivel/](Brilll)

**Comment by kivervinicius** [comment]
Cara eu corro, pedalo e tento manter uma dieta "saudável" mas não é facil principalmente quando chega em casa após as 18h vou correr chego umas 20h corpo ta destruído não da muita vontade de pegar o pc e ir fazer um freela ou estudar algo. Mas isso depende muito de cada pessoa, mas caso eu chegar e ir direto para o pc eu passo a noite direta..

**Reply by jucely** [reply]
Meu Deus, eu li que vc corre pelado. Precsiso de férias.

**Comment by felipefialho** [comment]
O maior especialista do mundo entre ser desenvolvedor e ter uma vida fitness é o @raphaelfabeni e com certeza ele pode falar melhor sobre isso.

Mas falando por mim, nunca deixei de fazer coisas que fazia antes de ser desenvolvedor:

- Jogar futebol
- Frequentar academia focado no minimo 3 vezes por semana
- Me alimentar bem e o mais corretamente possível

Uma sugestão interessante para ingressar e se acostumar com atividades fisicas é ir a pé para o trabalho, caso a distancia seja proibitiva, descer alguns pontos /  estação antes ou mesmo parar o carro mais longe é uma boa alternativa.

Na correria a alimentação também fica comprometida. Comum por exemplo, aquela semana com pizza todos os dias, lasanha pronta, coca-cola... cuidado com isso. Basta se policiar e trocar essas coisas por alimentos mais saudaveis, com certeza vai fazer bem.

**Comment by herberthenrique** [comment]
Quando comecei a trabalhar com desenvolvimento, eu achava que minha vida não permitia esse tipo de coisa. Eu era bem sedentário e dedicava meu tempo livre entre trabalhar e faculdade.
Depois da faculdade, apenas trabalhava mais do que antes e estudava outras tecnologias.

Chega um momento na sua vida em que você percebe que essa rotina te faz muito mal.
Eu nunca gostei de academia e então procurei algum esporte para sair da vida de sedentário.
Comecei a fazer boxe e exercícios físicos são ótimos para eliminar o estresse.

No fim, uma rotina e agendas são o que vão te ajudar a seguir uma vida mais saudável.
É só ter um pouco de força de vontade. :)

**Comment by raphaelfabeni** [comment]
Especialista nada @LFeh ! hahaha!
Mas vamos lá: nos últimos anos mantenho a mesma rotina, apenas fazendo alguns pequenos ajustes em relação a horário.  Atualmente em relação a alimentação e atividades ela engloba: 

* academia (6x semana)
* pedal (1 a 2x semana)
* futebol (agora esporádico, mas cheguei a jogar todo final de semana).
* acompanhamento nutricional

Pode parecer muita coisa, mas desde pequeno a prática de esportes estava presente no meu dia a dia, então atualmente é algo impossível de desapegar. Um outro ponto é que sou filho de professora, então desde pequeno gosto de estudar, então, meio que naturalmente consegui achar o meio termo entre as divisões: _estudo_ x _trabalho_ x _vida pessoal_ x _prática de esportes_. Acho que o ponto vital é você conhecer seu corpo e mente e saber quando é a hora de chegar em casa e simplesmente dormir, e desencanar do resto.

Concordo com o @herberthenrique que com o tempo, você precisa de um planejamento maior, principalmente dependendo dos seus objetivos, mas acredito que com motivação e um pouco de organização é possível sim iniciar um estilo de vida mais saudável (não vou mentir que no começo dá vontade de desistir, mas no fim vale a pena).

**Comment by bernardodiasc** [comment]
Acho que nosso ramo exige muito, há alguns anos aderi bons habitos e abandonei varios ruins. Esportes e alimentação saudável fazem parte da rotina hoje e embora tenha custado um pouco me adaptar no começo, agora faço tudo com muito gosto e satisfação, especialmente depois de sentir na pele os benefícios de fazer escolhas melhores pro corpo, pra mente e pra alma.

Além do básico musculação e cardio, recomendo muito yoga, meditação e muito alongamento.

Outra coisa, recentemente estive experimentando mesclar a técnica de produtividade Pomodoro com exercícios. O resultado foi que antes do meio dia eu ja tinha entregado varias tarefas, já tinha feito 100 de cada: abdominais, flexoes e agaixamentos, e alongamentos fortes, na metadade da tarde uma caminhada de leve de uns 30 min. Veja, trabalhei o dia todo e cuidei de mim, tudo tranquilamente. É possível, não digo que seja molezinha, mas é possível.

**Comment by marqueslu** [comment]
Acredito eu que o importante é separar as coisas e definir horários para cada tarefa. Não adianta nada ir na acadêmia mas estar pensando na issue ou PR que você abriu. 
Eu tenho uma rotina de treino de 5x semana. Consigo realizar os treinos no intervalo entre saída do escritório e ida pra faculdade.
Comecei a aderir esse hábito recentemente e está me fazendo muito bem.

**Comment by kvnol** [comment]
Além do desenvolvimento tenho mais duas paixões: **música** e **artes marciais** e posso te dizer que foram as melhores escolhas que fiz na minha vida.

* Pratico musculação 4/6 vezes por semana
* Muay Thai e Boxe (e se tudo der certo, vou começar a estudar para virar professor) duas vezes por semana
* Aula de canto uma vez por semana

Já cheguei a jogar futsal pra valer num time da cidade vizinha há alguns anos e estou parado do futebol desde o começo do ano quando trinquei o pulso (futebol é um esporte mais violento que Muay Thai), mas espero voltar logo e jogar com uma galera menos bruta.

E sobre a música, toco vários instrumentos mas o mais importante é minha aula de canto que não consigo deixar de fazer, pois além da minha paixão pela música que faria eu largar de tudo para viver dela, a saúde da voz e respiração é importantíssima (respirar fundo não é levantar os ombros).

**Comment by alinebastos** [comment]
Acho que depende de muitos fatores.
Quando eu trabalhava fora (presencial) eu caminhava bastante, escolhia pegar um ônibus ao invés de dois pra caminhar mais (e economizar).
Comecei a trabalhar remoto há uns 3 anos. Quase não saio mais de casa. Fico sentada o dia todo. Passo comendo o dia todo. Engordei, claro!
Estou "entre empregos" agora, hehe, aí quero ver se acho algo presencial, ou algo que pague bem pra eu ir pra um coworking.
Tem que ter disposição pra sair de casa se trabalha remoto.

**Comment by shadowmaru** [comment]
Quando eu morava perto do trabalho, eu ia todo dia de bicicleta. Daí a gente parte pra drogas mais fortes, como cicloviagens, ciclismo de longa distância...

Agora moro mais longe e fica mais difícil fazer isso, ainda mais no calor. Consigo fazer parcialmente, mas a motivação não é mais a mesma. Academia também é aquele começa e para. E olha que eu trabalho para a Smart Fit 😛 

Mas também já pratiquei vários esportes: karatê, natação, vôlei, beisebol, ciclismo.

**Comment by synga** [comment]
Acho que vai muito da motivação que nós temos mesmo (ou a falta dela). Por mais que tenham horas extras, novas coisas pra aprender, projetos pessoais e freelas sempre da pra arranjar um tempo, e não precisa ser 1h e meia de academia na manhã e 1 hora pedalando ou caminhando a noite todos os dias.

Qualquer exercício conta, 40min de caminhada todo dia já é o suficiente pra dar uma agitada boa no corpo. Então acho que da sim da pra ter uma vida bem saudável, o que falta é motivação nossa mesmo, ainda mais quando ficamos nessa pilha de que pra T.I. você tem que estudar e trabalhar 24/7. Além da caminhada, beber bastante água e comer bem faz toda a diferença.

Eu mesmo faço nada de exercício, faço 1 abdominal todo dia quando estou levantando da cama 😄 mas saio de casa pra ir pro trabalho as 7:45 e chego dele as 17:25, tenho livre por dia +/- 7h e uso isso apenas estudando em casa, sendo que conciliando bem dava pra fazer uma academia, estudar e ter um tempinho de lazer.

**Comment by sbvorpagel** [comment]
Uma dica é aderir hábitos saudáveis durante o seu dia. 

Ex.: 

Em vez de ir de carro, ônibus, metro, avião ou seja lá como for para o trabalho, tente ir caminhando/correndo ou de bike. No começo é complicado mas logo se acostuma.

Faça pausas e alongamentos curtos a cada hora no trabalho, nada exagerado, mas uma esticada entre um café e um código ajuda muito.

Faça academia. Academia te permite fazer uma carga de exercício boa em pouco tempo, e se precisar muito deste tempo, pode faze-la ouvindo um podcast ou áudio livro (mas recomendo também ter um tempo para fugir da computação)

Compre uma esteira adaptada para notebook e trabalhe igual o Linus Torvalds (Brincadeira)

**Comment by felipebernardes** [comment]
+1 pro @sbvorpagel 

Um adendo: faça de pouco em pouco. Rotina e hábitos saudáveis demandam tempo pra se estabelecer. 

Adicione um costume pequeno e significativamente saudável à sua rotina, pratique bastante e quando estiver acostumado, repita.

**Comment by Thentomint** [comment]
> Bom dia Pessoas tudo bem com você master dev ninjas, minha dúvida é o seguinte parace uma coisa tola, mais tive essa dúvida. Será que um desenvolvedor consegue ter uma vida Saudável com essa rotina diária na nossa área, é bem corrido por sinal com várias demandas, várias tecnologias que conseguimos aprender enfim, são muitas novidades no dia a dia.
> 
> Será que consegue ter um vida sei lá fitness kkkkk. ótimo dia a todos e um excelente final de semana para vocês!
> 
> Quem disse que programador não pode ter uma vida SAÚDAVEL! E no meu tempo livre, eu jogo [brasil roleplay](https://riorise.com/) é sempre uma experiência divertida!
> 
> Vi esse artigo falando sobre isso. [https://blog.umbler.com/br/vida-saudavel-para-desenvolvedores-e-possivel/](Brilll)

Cara, que tópico massa! Me identifiquei DEMAIS com o que você escreveu.

Tô no começo da carreira como dev, ainda tentando me encontrar entre tantas stacks, frameworks, prazos... é um corre danado. Mas sabe o que me salvou? Comecei a incluir pequenas pausas no dia pra alongar, caminhar um pouco, até levanto da cadeira só pra respirar na varanda haha. Pode parecer besteira, mas isso já mudou muito minha energia e até minha concentração no código.

Tô longe de ser “fitness” também kkkk, mas tô tentando manter o equilíbrio alimentação melhor, um treininho leve no fim do dia. Acho que o segredo é não buscar perfeição, mas constância.  Abraço e bom finde pra você também!
